ecs-cli/modules/clients/aws/cloudformation/params_test.go

```diff
@@ -59,8 +59,8 @@ func TestAddAndValidate(t *testing.T) {
 	}
 
 	paramsMap := cfnParams.Get()
-	if len(requiredParameterNames) != len(paramsMap) {
-		t.Errorf("Mismatch in number of keys in params map. %d != %d", len(requiredParameterNames), len(paramsMap))
+	if len(paramsMap) != 2 {
```

Comment:
What is the significance of 2? Is there a more descriptive way we can make this assertion?

Reply:
2 is just the number of parameters that were added in the test. So its checking if the 2 calls to the parameter add method actually added 2 parameters.

Reply:
I can make 2 a const with a descriptive name tho